app/controllers/barbeque/sns_subscriptions_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Barbeque::SnsSubscriptionsController < Barbeque::ApplicationController
+  def index
+    @sns_subscriptions = Barbeque::SNSSubscription.all
```

Comment:
I know that. But IMO it is useful for web browser's find-in-page feature as long as there are thousands records.